keyboards/dotmod/dymium65/rules.mk

```diff
@@ -0,0 +1,18 @@
+# Build Options
+#   change yes to no to disable
+#
+
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+MOUSEKEY_ENABLE = yes       # Mouse keys
+
+ENCODER_ENABLE = yes	    # Enable ENCODER
```

Comment:
If encoders are enabled at the keyboard level, there should be some encoder functionality in `dymium65.c` file using the `encoder_update_kb()` function, that way it works for QMK Configurator users.

Reply:
thank you for your help, we will modify it

Reply:
You have not.